sample/ios/RtnGimbalSdkSample/RtnGimbalSdkSample.entitlements

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Does this do anything yet, considering that we haven't implemented `CommunicationManager`?

Reply:
oops, this was a holdover from when Nick was asking about testing communicates, I'll remove it now and add it back in later, sounded like we're going to have more discussions surrounding CommunicationManager.